jvm-packages/pom.xml

```diff
@@ -301,14 +301,6 @@
                     <url>https://s3.amazonaws.com/xgboost-maven-repo/release</url>
                 </repository>
             </repositories>
-            <modules>
```

Comment:
Please, note, that this profile is only responsible for the repositories now. 
Release has to be done specifying profiles `default` and `gpu` **in** **addition** to this one